### DISCUSSION TITLE
Branding

### DISCUSSION THREAD

**Discussion by ayoolafelix** [DevsHelpDevs/DevsHelpDevs#9] [General]
Can someone send a brief that contains the name and stuffs that are necessary for the logo to communicate?

**Comment by escamoteur** [comment]
check
https://github.com/DevsHelpDevs/DevsHelpDevs
https://github.com/DevsHelpDevs/DevsHelpDevs/blob/master/user-specification.md

**Comment by escamoteur** [comment]
If more than one designer joins our group, please feel all welcome.
If you have time please create all a preliminary design and logo so we have a range to choose.

**Reply by StephenLadoye** [reply]
@ayoolafelix How are you brother? Kindly invite your friends or other designers to work with via Twitter. Thanks

**Reply by ayoolafelix** [reply]
Will do @StephenLadoye
